src/theme.js

```diff
@@ -30,5 +31,20 @@ export default {
       color: colors.darkGray,
       textDecoration: 'none'
     }
+  },
+  rsvpLink: {
```

Comment:
Hey!  Sorry for the delay getting to this PR.  I'd actually say rsvpLink maybe shouldn't go in the global theme.  The purpose of the global theme here is for reusable elements across components -- the idea being we should start with styles per component and then abstract out the common ones (similar to what we'd do with programming and classes).  In this case, since we are only using this style in one component, I'd define the whole style just in there.  Thanks!